axelrod/tests/unit/test_moran.py

```diff
@@ -330,6 +331,20 @@ def test_iter(self):
         mp = MoranProcess((p1, p2))
         self.assertEqual(mp.__iter__(), mp)
 
+    def test_population_plot(self):
+        # Test that can plot on a given matplotlib axes
+        axelrod.seed(15)
+        N = 5
+        players = [random.choice(axelrod.basic_strategies)() for _ in range(N)]
+        mp = axelrod.MoranProcess(players=players, turns=200)
+        mp.play()
+        fig, axarr = plt.subplots(2, 2)
+        ax = axarr[1, 0]
+        mp.populations_plot(ax=ax)
```

Comment:
Can we also have a test where we just call `mp.populations_plot()` (to test the default runs without a problem).